my-app/src/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+
+import Calculator from './component/Calculator';
+
+ReactDOM.render(
+  <Calculator />,
+  document.getElementById('root'),
+);
```

Comment:
- [Optional] I think it will be best practice if you use App.js here instead of index.js🤔